### ISSUE TITLE
Add py.typed to mark package as type annotated

### ISSUE THREAD

**Issue by HarrySky** [collerek/ormar#65]
Hi, thank you for your work on this nice ORM :smile:
Can you please add file `py.typed` to `ormar` folder?
I already see it is mentioned in setup.py `package_data={PACKAGE: ["py.typed"]}`, but no actual file is present, which causes mypy to give errors:

```
error: Skipping analyzing 'ormar': found module but no type hints or library stubs  [import]
```

**Comment by collerek** [comment]
Thanks for the nice word :)
Sure, will do in next release.

**Comment by collerek** [comment]
Should be closed in #66
